src/sdk/namespace_fs.js

```diff
@@ -723,9 +738,14 @@ class NamespaceFS {
                 }
             }
             await flush();
+            if (MD5Async) {
+                fs_xattr = this._assign_md5_to_fs_xattr((await MD5Async.digest()).toString('hex'), fs_xattr);
+            }
             if (fs_xattr) {
                 await target_file.setxattr(fs_account_config, fs_xattr);
             }
```

Comment:
After we done with `target_file` we should close it and make `target_file = null` and not count on the finally.